README.md

```diff
@@ -4,21 +4,21 @@ The valves and pump are unlocked by researching **Fluid handling**, the pipes ha
 ## Non-return valve
 - Also known as a check valve, this allows full-speed flow in one direction, indicated by the green arrow.
 - Can output its fluid level to red/green wires.
-- Recipe: 1 iron plate, 1 iron gear, 1 pipe
+- Recipe: <sup>1</sup> <img title="Iron plate" src="https://i.imgur.com/tUJWXWG.png" height="23px">, <sup>1</sup> <img title="Iron gear wheel" src="https://i.imgur.com/9cwwzNR.png" height="23px">, <sup>1</sup> <img title="Pipe" src="https://i.imgur.com/G5HG09q.png" height="23px">
```

Comment:
That is cool. But maybe lets not get it from imgur. It's possible to get it from the Factorio wiki. If you make that change we can merge it :)
```
![](https://wiki.factorio.com/images/thumb/Iron_plate.png/32px-Iron_plate.png)
```
![](https://wiki.factorio.com/images/thumb/Iron_plate.png/32px-Iron_plate.png)